packages/app/src/app/components/CreateSandbox/Import/Import.tsx

```diff
@@ -58,6 +63,7 @@ export const Import: React.FC<ImportProps> = ({ onRepoSelect }) => {
   const { hasLogIn, activeTeam } = useAppState();
   const importAndRedirect = useImportAndRedirect();
 
+  const ghAuth = useGitHubAuthorization();
```

Comment:
We can destructure `allowsPublicRepos` here

Reply:
https://github.com/codesandbox/codesandbox-client/pull/7385/commits/e6b22d0e4e3f5e72d5af2aef96e4e1a06a67d31d